metrics-report-generator/src/main/scala/it/pagopa/interop/metricsreportgenerator/util/models.scala

```diff
@@ -129,7 +175,8 @@ object Report {
     (aId, pId, time)
   }
 
-  private val header: String = "agreementId,purposeId,year,month,day,tokencount"
+  private val headerCsv: String         = "agreementId,purposeId,year,month,day,tokencount"
+  private val headerSheet: List[String] = List("agreementId", "purposeId", "year", "month", "day", "tokencount")
```

Comment:
To reduce maintenance mistakes
```suggestion
  private val headerSheet: List[String] = List("agreementId", "purposeId", "year", "month", "day", "tokencount")
  private val headerCsv: String         = headerSheet.mkString(",")
```